src/content/releases/_index.en.md

```diff
@@ -5,6 +5,20 @@ pre = "<b>6. </b>"
 weight = 30
 +++
 
+## v0.7.0 StatefulSet support for service discovery and benchmark tooling
+
+> This release mainly focused on adding support for StatefulSets in Lighthouse for service discovery and adding new `subctl`
+> commands to benchmark the network performance across clusters.
+
+* **Lighthouse** enhancements/changes:
```

Comment:
```suggestion
* Lighthouse enhancements/changes:
```

Reply:
I did that b/c we bold it in other places. I should change the other places to be consistent.